cmd/anubis/main.go

```diff
@@ -404,6 +405,10 @@ func main() {
 		slog.Warn("generating random key, Anubis will have strange behavior when multiple instances are behind the same load balancer target, for more information: see https://anubis.techaro.lol/docs/admin/installation#key-generation")
 	}
 
+	if customRealIPHeader != nil && *customRealIPHeader == "" {
+		log.Fatalf("custom-real-ip-header option requires a value")
+	}
+
```

Comment:
```suggestion
```

The if predicate always evaluates to true, flag.String allocates a pointer to empty string by default.